mobile/build.gradle

```diff
@@ -140,7 +144,7 @@ dependencies {
     compile "info.metadude.android:typed-preferences:2.0.0"
     compile "org.jboss.aerogear:aerogear-android-authz:3.0.0"
     compile "org.jboss.aerogear:aerogear-android-pipe:3.1.0"
-    compile "org.jboss.aerogear:aerogear-android-push:3.0.0"
+    compile "org.jboss.aerogear:aerogear-android-push:4.0.0"
```

Comment:
Not confirmed if issue I faced was because of 4.1.x. Updating to 4.0.2 Will update to 4.1.x later after surety